packages/blade/private/client/utils/data.ts

```diff
@@ -24,7 +123,20 @@ export const fetchRetry = async (
 ): Promise<Response> => {
   return retry(async (bail) => {
     try {
-      const response = await fetch(request, requestInit);
+      // Note: Safari has an issue where streams are closed but the connection remains open
+      // so UI flushing / revalidation will end up flickering between the current and previous
+      // page.
```

Comment:
```suggestion
      // Safari has an issue where streams are closed but the connection remains open, so
      // UI flushing / revalidation will end up flickering between the current and previous page.
```

Avoids the dangling word at the end.